tests/reading-tests.md

```diff
@@ -295,68 +294,89 @@ Drawing also from the first unit test, Leila concludes:
 
 To summarize, this second test expects the value of `fizzBuzz(25)` to equal `'Buzz'`.
 
-## Example on Replit.com
+## Example on CodeSandbox.io
+
+We can run a version of this test suite on [this CodeSandbox project](https://codesandbox.io/s/fizzbuzz-with-tests-488sq?file=/test/fizzbuzz.test.js).
+
+CodeSandbox.io is a platform for creating projects. We'll use this platform to read, write, and run our FizzBuzz unit tests.
+
```

Comment:
suggest a callout to give the reader a chance to orient themselves in the new space, to recognize what's similar, and what's different, and to set the tone that this is something we'll do often as we move through different environments.
```suggestion

### !callout-info

## CodeSandbox Is the Same, but Different

CodeSandbox provides an environment for editing code, just like Replit.com or even VS Code do. We should spend a few minutes looking around to identify where we can find some of the common features in any IDE. Some areas to keep an eye out for include:
- a list of files in the project
- a file editor view
- search tools
- output views
- configuration settings

<br />

We should feel comfortable exploring this environment until we feel like we have a general idea of where our main project areas are.

### !end-callout

```